src/WebApplication/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+﻿{
```

Comment:
Are we supposed to add this to source control? (remember tht IIS Express means that shutdown doesn't work)